smartclide-monitoring/src/main/java/eu/smartclide/contexthandling/services/GitlabApiClient.java

```diff
@@ -123,8 +123,15 @@ public List<GitlabCommitMessage> getGitlabCommitMessages() {
     private int calculateTimeSinceLastCommit(String projectId, JsonObject commit) {
         int difference = 0;
         // check if parent id exists for given commit
-        if (commit.get("parent_ids").getAsJsonArray().size() > 0) {
-            String parentCommitId = commit.get("parent_ids").getAsJsonArray().get(0).getAsString();
+        JsonArray parentIds = commit.get("parent_ids").getAsJsonArray();
+        if (parentIds.size() > 0) {
+            logger.info("-----------------Found {} parent_ids with commit id {}", parentIds.size(), commit.get("id").getAsString());
+            logger.info("parent_ids: {}", parentIds);
```

Comment:
Combine the two log statements into one. And remove the `-----------------`.
It's useful for debugging purposes but should not be necessary any more.